src/main/kotlin/com/msd/robot/domain/Robot.kt

```diff
@@ -145,17 +149,9 @@ class Robot(
     }
 
     fun attack(otherRobot: Robot) {
-        try {
```

Comment:
Why was the error handling removed? We still have to check whether a robot has enough energy to attack another robot.